composer.json

```diff
@@ -11,7 +11,7 @@
 		}
 	],
 	"require": {
-		"php": ">= 7.1.0"
+		"php": "^7.4 || ^8.0"
```

Comment:
Dajme sem < 8.2, keďže zatiaľ nemáme istotu že to pojde a po vydaní verzie sa to už nedá zvratit